doc/setup/install.rst

```diff
@@ -213,23 +272,42 @@ easily keeping (and running) multiple versions of Odoo side-by-side.
 There are two way to get the odoo source source zip or git.
 
 * Odoo zip can be downloaded from
-  https://nightly.odoo.com/9.0/nightly/src/odoo_9.0.latest.zip, the zip file
-  then needs to be uncompressed to use its content
+  https://nightly.odoo.com/9.0/nightly/src/odoo_9.0.latest.zip or the Download_
```

Comment:
while you are adding link, replace this hardcoded link by something like [direct link](https://nightly.odoo.com/9.0/nightly/src/odoo_9.0.latest.zip)